File: Universal Windows Platform Solution/Phantom of the West/Phantom of the West/Visual Novel/Chapters/Chapter 5/IKickCheerleader.cs
using Phantom_of_the_West.Visual_Novel.Serialization.Event_Flagging;

namespace Phantom_of_the_West.Visual_Novel.Chapters.Chapter_5
{
	internal class IKickCheerleader : StoryChoice
	{
		internal IKickCheerleader() : base(StoryChoice.GetLocalizedString("ch5IKickCheerleader"), 5071)
		{
		}

		public override void Select()
		{
			PotWVN vn = PotWVN.MainVN;
			vn.EventFlags.Ch5SparredWithCheerleader = CheerleaderSpar.Kicked;
			base.Select();
		}
	}
}

File: Android Project/app/src/main/java/com/christopherraleigh/phantomofthewest/visual_novel/chapters/chapter_5/IKickCheerleader.java
package com.christopherraleigh.phantomofthewest.visual_novel.chapters.chapter_5;

import com.christopherraleigh.phantomofthewest.R;
import com.christopherraleigh.phantomofthewest.visual_novel.PotWVN;
import com.christopherraleigh.phantomofthewest.visual_novel.chapters.StoryChoice;
import com.christopherraleigh.phantomofthewest.visual_novel.serialization.event_flagging.CheerleaderSpar;

/**
 * Created by christopherraleigh on 2016-05-18.
 */
public class IKickCheerleader extends StoryChoice {
    public IKickCheerleader() {
        super(R.string.ch5IKickCheerleader, 5071);
    }

    @Override
    public void select()
    {
        PotWVN vn = PotWVN.getMainVN();
        vn.getEventFlags().setCh5SparredWithCheerleader(CheerleaderSpar.Kicked);
        super.select();
    }
}
